openfisca_us/parameters/gov/states/ut/tax/income/rate.yaml

```diff
@@ -1,6 +1,10 @@
 description: Utah Tax Rate
 values:
-  2021-01-01:
+  2008-01-01:
+    value: 0.0500
```

Comment:
@nikhilwoodruff. Check this. Does this strategy work better and is the indenting correct?

Reply:
Indenting must be wrong because the following CI errors are coming up now `E   Found duplicate key 'metadata'` and didn't prior to this change.
```
E   openfisca_core.errors.parameter_parsing_error.ParameterParsingError: Traceback (most recent call last):
E     File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/openfisca_core/parameters/helpers.py", line 44, in _load_yaml_file
E       return config.yaml.load(f, Loader = config.Loader)
E     File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/yaml/__init__.py", line 81, in load
E       return loader.get_single_data()
E     File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/yaml/constructor.py", line 51, in get_single_data
E       return self.construct_document(node)
E     File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/yaml/constructor.py", line 55, in construct_document
E       data = self.construct_object(node)
E     File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/yaml/constructor.py", line 100, in construct_object
E       data = constructor(self, node)
E     File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/openfisca_core/parameters/config.py", line 41, in dict_no_duplicate_constructor
E       return loader.construct_mapping(node, deep)
E     File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/yaml/constructor.py", line 218, in construct_mapping
E       return super().construct_mapping(node, deep=deep)
E     File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/yaml/constructor.py", line 143, in construct_mapping
E       value = self.construct_object(value_node, deep=deep)
E     File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/yaml/constructor.py", line 100, in construct_object
E       data = constructor(self, node)
E     File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/openfisca_core/parameters/config.py", line 39, in dict_no_duplicate_constructor
E       raise yaml.parser.ParserError('', node.start_mark, f"Found duplicate key '{duplicate}'")
E   yaml.parser.ParserError: 
E     in "/home/runner/work/openfisca-us/openfisca-us/openfisca_us/parameters/gov/states/ut/tax/income/rate.yaml", line 3, column 3
E   Found duplicate key 'metadata'
E   
E   Error parsing parameter file '/home/runner/work/openfisca-us/openfisca-us/openfisca_us/parameters/gov/states/ut/tax/income/rate.yaml':
E   Invalid YAML. Check the traceback above for more details.
...
E   yaml.parser.ParserError: 
E     in "/home/runner/work/openfisca-us/openfisca-us/openfisca_us/parameters/gov/states/ut/tax/income/rate.yaml", line 3, column 3
E   Found duplicate key 'metadata'
```

Reply:
Sorry I missed this! Resolving since this is fixed.